use_cases/init-process-for-fluent-bit/README.md

```diff
@@ -0,0 +1,227 @@
+
+Init process is designed to solve above mentioned problems and improve user experience of using Fluent Bit on ECS. You only need to set the `ARN` or `Path` of the config file in the ECS Task Definition, without telling us if it is a parser file. Init process will process each config file, use [@INCLUDE](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/configuration-file#config_include_file-1) keyword to add them to the main config file, and change the Fluent Bit command if the parser is used.
+
+The init process also injects ECS Task Metadata into the Fluent Bit container as environment variables, so you can use them in the Fluent Bit config.
+
+
+
+### How to use multi-config feature?
+
+1. Use `aws-for-fluent-bit` images with `init` tag.
+
+2. Specify config files as environment variable in the FireLens configuration Environment setting.
+
+   **Example:**
+
+   ```
+   "environment": [
+                   {
+                       "name": "aws_fluent_bit_init_s3_1",
+                       "value": "arn:aws:s3:::yourBucket/aaaaa.conf"
+                   },
+                   {
+                       "name": "aws_fluent_bit_init_s3_2",
+                       "value": "arn:aws:s3:::yourBucket/bbbbb.conf"
+                   },
+                   {
+                       "name": "aws_fluent_bit_init_file_1",
+                       "value": "/ecs/s3.conf"
+                   }
+               ]
+   ```
+
+   **We support two kinds of config files:**
+
+   * Config files which stored in your S3 bucket
+     You need to set the name of the env var using prefix `aws_fluent_bit_init_s3_`, the number after this prefix cannot be repeated, and set the `ARN` of your config file as the value of the env var.
+
+   * Config files which from the image
+
+     * Use [our built-in config files](https://github.com/aws/aws-for-fluent-bit/tree/mainline/ecs)
+     
+       You need to set the name of the env var using prefix `aws_fluent_bit_init_file_`, the number after this prefix cannot be repeated, and set the `Path` of your config file as the value of the env var.
+     
+     * Build a custom image
+     
+       You can build a custom image using image with `init` tag as the base and add your own configs. For more details please see [this example](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/config-file-type-file).
+     
+
+3. Set up your ECS Task role
+
+   If you specify the config file from your S3 bucket, the init process will download it from your bucket. So please add the following permissions: 
```

Comment:
since folks always get confused about hte roles, lets add a link to here: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html

And note that this is different than the task execution role.